.github/workflows/ci.yml

```diff
@@ -72,13 +72,13 @@ jobs:
           mkdir build
           mv dist-storybook $PR_NUMBER
           cp -R $PR_NUMBER build/
-        if: github.ref != 'refs/heads/main'
+        if: github.ref != 'refs/heads/main' || github.repository == github.event.pull_request.head.repo.full_name
```

Comment:
Surely we should check the build works even if we won't deploy it?

Reply:
This is already done in a previous step as if you look at this step its preparing to be deployed not building and just copies files from one directory to another

Reply:
Should this logic be `branch is not main AND is PR from repo`?

Reply:
Ah yes good shout!